src/Component/Calculator.js

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
 import './Calculator.css';
+import PropTypes from 'prop-types';
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ OPTIONAL ] Kindly don't disable the linters since it is here to make your code better so you should solve the issues to have a clean and dry code ✅👌🏻